core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -516,6 +531,25 @@ export class NavigationComponent {
       });
   }
 
+  private listenToLockChange(): void {
+    this.workflowCollabService
+      .getLockStatusStream()
+      .pipe(untilDestroyed(this))
+      .subscribe((isLockGranted: boolean) => {
+        if (isLockGranted) this.lockGranted = true;
+        else this.lockGranted = false;
+      });
+  }
+
+  private listenToWorkflowAccess(): void {
+    this.workflowCollabService
+      .getWorkflowAccessStream()
+      .pipe(untilDestroyed(this))
+      .subscribe((isWorkflowReadonly: boolean) => {
+        this.isWorkflowReadonly = isWorkflowReadonly;
```

Comment:
ditto, please consider removing the `is` in the varible name.

Reply:
Done.